src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -1142,7 +1143,8 @@ protected function getReferencedId($value, ResourceFieldInterface $resource_fiel
         $target_entity_type = 'taxonomy_vocabulary';
       }
     }
-    elseif (!empty($field_info['type']) && $field_info['type'] == 'entityreference') {
+    elseif (!empty($field_info['type']) 
```

Comment:
I don't think this is correct. This will prevent the next `elseif` from being executed.